### PR TITLE
Initial Load

### PR DESCRIPTION
**Pull request by torstenboettjer**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+[cols="a,a",options="header,autowidth"]
+|===
+|Name |Description
+|[[output_compartment_id]] <<output_compartment_id,compartment_id>> |OCID for the network compartment
+|[[output_vcn_id]] <<output_vcn_id,vcn_id>> |Identifier for the Virtual Cloud Network (VCN)
+|[[output_gateways]] <<output_gateways,gateways>> |A list of gateways for the Virtual Cloud Network (VCN)
+|[[output_subnets]] <<output_subnets,subnets>> |A list of subnets for the Virtual Cloud Network (VCN)
+|[[output_route_tables]] <<output_route_tables,route_tables>> |A list of route_tables for the Virtual Cloud Network (VCN)
+|[[output_security_lists]] <<output_security_lists,security_lists>> |All security lists defined for the Virtual Cloud Network (VCN)
+|===
```
Comment (VetteByte) [inline README.adoc:162]: Please add the license/disclaimer text here (copy-and-paste from the MD version of the README).

File: resources.tf [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    ]
+    compartment_id = data.oci_identity_compartments.network.compartments[0].id
+    vcn_id         = oci_core_vcn.segment.id
+    for_each       = var.network.subnets
+    cidr_block     = each.value.cidr_block
+    display_name   = each.value.display_name
+    dns_label      = each.value.dns_label
+    defined_tags   = var.assets.resident.defined_tags
+    freeform_tags  = var.assets.resident.freeform_tags
+    #route_table_id = local.route_tables[each.value.route_table]
```
Comment (VetteByte) [inline resources.tf:119]: Should this be uncommented?

**Review by VetteByte — CHANGES_REQUESTED** [review]
Everything looks good except for the README (please add the license/disclaimer section).

File: README.adoc (hunk 1 above)
Comment (torstenboettjer) [reply README.adoc:162]: Done

File: resources.tf (hunk 2 above)
Comment (torstenboettjer) [reply resources.tf:119]: Done by purpose, this will become a conditional entry in the next version.

**Review by torstenboettjer — COMMENTED** [review]
done

**Review by VetteByte — APPROVED** [review]
Great work!

**Pull request merged**